<launch>
	<node name="recognize_face_node" pkg="vision" type="recognize_face_node.py">
	</node>
	<node name="learn_face_node" pkg="vision" type="learn_face_node.py">
<!--
		<param name="in" value="/unrecognized_face"/>
		<param name="out" value="/learned_face"/>
		<param name="delta_xy_px" value="20"/>
		<param name="delta_t_ms" value="1000"/>
		<param name="facedb" value="facedb"/>
		<param name="min_faces" value="5"/>
		<param name="max_faces" value="10"/>
-->
	</node>
</launch>
